File: api/src/enums.py
from enum import Enum


class PictureCategory(str, Enum):
    AVATAR = 'avatar'
    PAIRED_AVATARS = 'paired_avatars'
    CUTE = 'cute'
    ANGRY = 'angry'


class ChatType(str, Enum):
    PRIVATE = 'private'
    GROUP = 'group'
    SUPERGROUP = 'supergroup'

File: api/enums.ts
// from enum import StrEnum

// from i_texts import texts as t

export type Command =
  | "start"
  | "get_avatars"
  | "get_paired"
  | "get_cute"
  | "get_angry"
  | "send_picture"
  | "admin"

export type PictureCategory =
  | "avatar"
  | "paired_avatars"
  | "cute"
  | "angry"
